docker-compose.yml

```diff
@@ -1,11 +1,66 @@
 version: '3'
 
 services:
-  web:
-    build: .
+
+  reverseproxy:
+    build:
+      context: ./reverse-proxy
+      dockerfile: ./Dockerfile
     ports:
-      - "4000:4000"
+      - 80:80
+      - 443:443
+    restart: unless-stopped
     volumes:
-      - .:/app
+      - ./certbot/conf:/etc/letsencrypt
+      # - ./certbot/www:/var/www/certbot # only if you want to use the webroot authenticator
     env_file:
       - .env
+    networks:
+      - jellyfish_network
+
+  frontend:
+    build:
+      context: ./assets
+      dockerfile: ./Dockerfile
+    restart: unless-stopped
+    networks:
+      - jellyfish_network
+
+  backend:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    environment:
+      SECRET_KEY_BASE: ${SECRET_KEY_BASE}
+      JELLYFISH_IP: "jellyfish"
+      JELLYFISH_PORT: 5002
+      JELLYFISH_API_TOKEN: ${SERVER_API_TOKEN}
+      PHX_SERVER: true
+      PHX_HOST: "${DOMAIN}"
+    restart: unless-stopped
+    networks:
+      - jellyfish_network
+
+  jellyfish:
+    image: "ghcr.io/jellyfish-dev/jellyfish:${TAG:-edge}"
+    container_name: jellyfish
+    restart: unless-stopped
+    environment:
+      VIRTUAL_HOST: "${DOMAIN}"
+      USE_INTEGRATED_TURN: "true"
+      INTEGRATED_TURN_IP: "${INTEGRATED_TURN_IP}"
+      INTEGRATED_TURN_LISTEN_IP: "0.0.0.0"
+      INTEGRATED_TURN_PORT_RANGE: "50000-50050"
```

Comment:
We should probably use more ports than that - I think a value of 500 ports (which corresponds to maximum of 500 peers being able to connect at the same time) should be fine.

Reply:
ok